decoder/routes.go

```diff
@@ -0,0 +1,205 @@
+package decoder
+
+import (
+	"database/sql"
+	"encoding/json"
+	"fmt"
+	"github.com/jellydator/ttlcache/v3"
+	"golbat/db"
+	"golbat/pogo"
+	"time"
+)
+
+type Route struct {
+	Id               string  `db:"id"`
+	Name             string  `db:"name"`
+	Description      string  `db:"description"`
+	DistanceMeters   int64   `db:"distance_meters"`
+	DurationSeconds  int64   `db:"duration_seconds"`
+	EndFortId        string  `db:"end_fort_id"`
+	EndImage         string  `db:"end_image"`
+	EndLat           float64 `db:"end_lat"`
+	EndLon           float64 `db:"end_lon"`
+	Image            string  `db:"image"`
+	ImageBorderColor string  `db:"image_border_color"`
+	Reversible       bool    `db:"reversible"`
+	StartFortId      string  `db:"start_fort_id"`
+	StartImage       string  `db:"start_image"`
+	StartLat         float64 `db:"start_lat"`
+	StartLon         float64 `db:"start_lon"`
+	Tags             *string `db:"tags"`
```

Comment:
Should this be null.String ?

Reply:
Shit, I knew there was a way to handle this